sys/include/net/gnrc/netif/hdr.h

```diff
@@ -70,6 +70,20 @@ extern "C" {
  *          this flag the same way it does @ref GNRC_NETIF_HDR_FLAGS_BROADCAST.
  */
 #define GNRC_NETIF_HDR_FLAGS_MULTICAST  (0x40)
+
+/**
+ * @brief   More data will follow
+ *
+ * This flag signals that this packet is part of a burst of packets. The link
```

Comment:
Comparing to the other flags above, shouldn't this go under `@details`?

Reply:
It's redundant, details is implicit when a new paragraph is started after the brief description.

Reply:
Anyhow, for consistency reasons I would leave it in. Or remove the others, but that shouldn't be part of this PR I guess...